.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: required
 
 language: ruby
 rvm:
-  - 2.2
-  - 2.3.0
+  - 2.3.3
```

Comment:
I would keep 2.2 for now... we have intentions to drop 2.2 in manageiq soon but we're not there yet.

See: http://talk.manageiq.org/t/adding-support-for-ruby-2-4-and-dropping-2-2/2305